downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -68,8 +70,37 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### Ballerina Profiler (experimental)
+
+Introduced the `profile` CLI command, which runs a Ballerina package and does a CPU profile of it during runtime.
+
+```
+$ bal profile
+```
+
+- For example, if we run the above command in the root directory of a Ballerina package, it generates a flame graph that shows the time taken to execute each function.
+
+- The output is given by the `ProfilerOutput.html` file, which can be opened using a web browser.
+
+>**Note:** This is an experimental feature, which supports only a limited set of functionality.
+
 ### Improvements
 
+#### Java 17 support
+
+Ballerina now supports code compilation and execution with Java 17.
+
+#### Support large list and mapping constructors
+
+The number of members supported in a list constructor expression and the number of fields supported in a mapping constructor expression have been increased to create new array, tuple, map, and record values that are larger in size.
```

Comment:
``` suggestion
The number of members supported in a list constructor expression and the number of fields supported in a mapping constructor expression has been increased to create new values of array, tuple, map, and record that are larger in size.
```

Reply:
The correct grammar is to have `have` here. Regarding the second suggestion I think it is better to have the original or we can write it as `"to create new arrays, tuples, maps, and records that are larger in size"`.

Reply:
+1.

Reply:
Sorry, my bad.